src/main/java/io/jenkins/docker/client/DockerAPI.java

```diff
@@ -236,34 +240,25 @@ public void reallyClose() throws IOException {
     }
 
     /**
-     * Creates a new {@link DockerClient}.
-     * It's the caller's responsibility to dispose of the result.
+     * Creates a new {@link DockerClient}. It's the caller's responsibility to
+     * dispose of the result.
      */
     @SuppressWarnings("resource")
     private static SharableDockerClient makeClient(final String dockerUri, final String credentialsId,
             final Integer readTimeoutInMillisecondsOrNull, final Integer connectTimeoutInMillisecondsOrNull) {
-        NettyDockerCmdExecFactory cmdExecFactory = null;
         DockerClient actualClient = null;
         try {
-            cmdExecFactory = new NettyDockerCmdExecFactory()
-                    .withReadTimeout(readTimeoutInMillisecondsOrNull)
-                    .withConnectTimeout(connectTimeoutInMillisecondsOrNull);
-            final DefaultDockerClientConfig.Builder configBuilder = new DefaultDockerClientConfig.Builder()
-                    .withDockerHost(dockerUri)
-                    .withCustomSslConfig(toSSlConfig(credentialsId));
-            actualClient = DockerClientBuilder.getInstance(configBuilder)
-                .withDockerCmdExecFactory(cmdExecFactory)
-                .build();
+            DockerHttpClient httpClient = new ApacheDockerHttpClient.Builder().dockerHost(URI.create(dockerUri))
+                    .sslConfig(toSSlConfig(credentialsId)).build();
+            actualClient = DockerClientBuilder.getInstance().withDockerHttpClient(httpClient).build();
```

Comment:
Please retail the same "code style" as the previous code.

FYI the reason why it was putting each bit on its own line is so that, in future, if we need to add/remove/change any calls to the builder, all we're doing is adding/removing/changing one line - all the other lines will remain unchanged, and this makes viewing the history in git (e.g. using "git blame") _so_ much easier as you can see where each builder parameter came from.
If we allow multiple calls per line, it means that to add an extra "with..." the code change will show that we've rewritten the entire builder clause rather than just added one extra thing.

TL;DR: Multi-line builder code makes for more maintainable code.